src/store/modules/auth.js

```diff
@@ -53,147 +37,117 @@ export const getters = {
   userId: state => state.user && state.user.id,
   status: state => state.status,
   redirectTo: state => state.redirectTo,
+  ...metaStatuses(['login', 'update', 'changePassword', 'changeEmail']),
 }
 
 export const actions = {
+  clearLoginStatus: ({ dispatch }) => dispatch('meta/clear', ['login']),
```

Comment:
Maybe a helper for these would be useful, I was triggering root actions `groups/meta/clear` but I think having the actions here would be better.

Reply:
Let's see how much we actually need these `clear` actions, I wouldn't add a helper just for the sake of it. When we have a wrapper like this, it should be easy to refactor the modules.